### PR TITLE
document new connection/farm storage location

### PR DESCRIPTION
**Pull request by Luap99**
The new file was not really documented, so leave some pointers on how it works and that the new file should not be edited manually.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by mheon** [comment]
Sure, LGTM

File: docs/source/markdown/podman-system-connection.1.md [hunk 1]
```diff
@@ -11,6 +11,17 @@ Manage the destination(s) for Podman service(s).
 
 The user is prompted for the ssh login password or key file passphrase as required. The `ssh-agent` is supported if it is running.
 
+Podman manages the system connection by writing and reading the `podman-connections.json` file located under
```
Comment (rhatdan) [inline docs/source/markdown/podman-system-connection.1.md:14]: These should probably be handled via the options subdir, to make sure they don't drift.

Comment (Luap99) [reply docs/source/markdown/podman-system-connection.1.md:14]: AFAIK the options syntax is for cli options cannot be used here.

**Comment by Luap99** [comment]
@containers/podman-maintainers PTAL

File: docs/source/markdown/podman-system-connection.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/source/markdown/podman-system-connection.1.md:14]: @edsantiago Any reason this would not work?

**Review by flouthoc — APPROVED** [review]
LGTM

**Pull request merged**